darts/models/forecasting/forecasting_model.py

```diff
@@ -31,10 +31,12 @@
 
 import numpy as np
 import pandas as pd
+from sklearn.multioutput import MultiOutputRegressor
```

Comment:
not optimal to import this in the base classes. I think we can adapt the fit/predict wrappers as described in one of the comments